common/test/acceptance/pages/lms/learner_profile.py

```diff
@@ -43,7 +43,10 @@ def is_browser_on_page(self):
         """
         Check if browser is showing correct page.
         """
-        return self.q(css='body.view-profile .account-settings-container').present
+        return all([
+            self.q(css='body.view-profile .account-settings-container').present,
+            not self.q(css='ui-loading-indicator').visible
```

Comment:
@benpatterson I wasn't sure, so I tried it out. Tests with that page all errored with "Timed out waiting to load page". I could trace through the code, but my guess is that the class must get removed from the element or something, so "not visible" is what we want rather than invisible (present but not visible).